pkg/kv/kvclient/kvcoord/dist_sender.go

```diff
@@ -2198,7 +2198,7 @@ func skipStaleReplicas(
 	if !routing.Valid() {
 		return noMoreReplicasErr(
 			ambiguousError,
-			errors.Newf("routing information detected to be stale; lastErr: %s", lastErr))
+			errors.Wrap(lastErr, "routing information detected to be stale"))
```

Comment:
need to check with nathan whether the original code was intentional. we have historically handled this "lastErr" stuff specially.

Reply:
@nvanbenschoten friendly ping if you have time to look at the question above